src/libs/OptionsListUtils.ts

```diff
@@ -962,7 +962,7 @@ function createOption(
         const chatReport = allReports?.[`${ONYXKEYS.COLLECTION.REPORT}${report.chatReportID}`];
         const oneTransactionThreadReportID = getOneTransactionThreadReportID(report, chatReport, allReportActions?.[`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${report.reportID}`]);
         const oneTransactionThreadReport = allReports?.[`${ONYXKEYS.COLLECTION.REPORT}${oneTransactionThreadReportID}`];
-        result.isUnread = isUnread(report, oneTransactionThreadReport);
+        result.isUnread = isUnread(report, oneTransactionThreadReport, !!reportNameValuePairs?.private_isArchived);
```

Comment:
NAB I think this should be `!!result.private_isArchived` so we can reuse the same reference that was created earlier.